pem/lib/pem/options.rb

```diff
@@ -18,9 +18,16 @@ def self.available_options
                                      description: "Generate a p12 file additionally to a PEM file",
                                      is_string: false,
                                      default_value: true),
+        FastlaneCore::ConfigItem.new(key: :active_days_limit,
+                                     env_name: "PEM_ACTIVE_DAYS_LIMIT",
+                                     description: "If the current certificate is active for less than this number of days, generate a new one. Default value is 30 days",
+                                     default_value: 30,
+                                     verify_block: proc do |value|
+                                       UI.user_error!("Value of active_days_limit must be a positive integer or left blank") if value.to_s !~ /\A\d*\z/
```

Comment:
Can we check here `if value <= 0` instead of doing the string conversion and comparing against a regex?